src/components/ProblemSetBlock.tsx

```diff
@@ -32,6 +33,7 @@ export const NO_MORE_ATTEMPTS_MESSAGE = 'No more attempts allowed'
 export const PROBLEM_TYPE_INPUT = 'input'
 export const PROBLEM_TYPE_DROPDOWN = 'dropdown'
 export const PROBLEM_TYPE_MULTISELECT = 'multiselect'
+export const PROBLEM_TYPE_MULTIPLECHOICE = 'multiplechoice'
```

Comment:
Can we include a check similar to what's there for multiselect and dropdown [here](https://github.com/openstax/k12-apps-raise/blob/6c7c3d02a0e25fb0e6800800c87c737389a5d300/src/lib/blocks.tsx#L220) where we validate `data-solution-options` is provided if the problem type is `PROBLEM_TYPE_MULTIPLECHOICE`?